scripts/compute_hardnegs.py

```diff
@@ -55,7 +62,7 @@
         filtered_dataset,
         batch_size=8,
         shuffle=False,
-        collate_fn=lambda x: process_images(processor, [a["image"] for a in x]),
+        collate_fn=lambda x: processor.process_images(processor, [a["image"] for a in x]),
```

Comment:
Not maintained here (my fault) but this is not the terminology anymore (no need to pass processor)